Trimble.Modus.Components/Controls/TMSwitch/TMSwitch.xaml.cs

```diff
@@ -169,7 +221,8 @@ private static void OnSwitchSelected(TMSwitch tMSwitch)
                 { 0, 1, new Animation(v => tMSwitch.circle.TranslationX = v, tMSwitch.circleMargin,tMSwitch.border.WidthRequest - tMSwitch.circle.WidthRequest - tMSwitch.circleMargin) }
             },
             length: 250, easing: Easing.CubicIn);
-        tMSwitch.border.Color = ResourcesDictionary.ColorsDictionary(ColorsConstants.BlueLight);
+        VisualStateManager.GoToState(tMSwitch, "Selected");
+        UpdateSwitchStyle(tMSwitch);
```

Comment:
I don't think you have to update the style everytime the state changes. Can you remove this and check?

Reply:
Yes. It wasn't necessary. I removed it.Yes. It wasn't necessary. I removed it.